css/style.css

```diff
@@ -1,18 +1,522 @@
-.partners-list {
-	width: 100%;
-	margin-bottom: 20px;
-	border: 1px solid #000000;
-	border-collapse: collapse; 
-	background-color: #7de0d3;
+
+body{
+	font-family: "Bakbak One", "Roboto", sans-serif;
+	font-weight: 400;
+	font-style: normal;
+	color: #fff;
+	background-color: #1E1E1E;
+}
+
+a{
+  text-decoration: none;
+  color: currentColor;
+  text-transform: uppercase;
+}
+
+ul,
+ol{
+  list-style: none;
+  color: currentColor;
+}
+
+h1,
+h2,
+h3{
+  color: currentColor;
+  text-transform:uppercase;
+}
+
+h1{
+  font-size: 72px;
+}
+
+h2{
+  font-size: 44px;
+  line-height:56px;
+}
+
+h3{
+  font-size: 24px;
+}
+
+header{
+  text-transform: uppercase;	  
+}  
+
+.hidden{
+  display: none;
+}
+
+.page-header{
+  background-color: #040B11;
+  display: inline-flex;
+}
+
+.logo{
+  display: inline-flex;
+  font-size: 32px;
+  line-height: 42px;
+  gap: 16px;
+}
+
+.header-nav-list{
+  display: flex; 
+  font-size: 16px; 
+  line-height: 24px;
+  letter-spacing: 0.1em;
+  gap: 32px;
+}
+
+.button-theme-toggle{ 
+  width: 48px;
+  height: 48px;
+  border: none;
+  border-radius: 41px;
+  background-color: #141B22;
+}
+
+.button-action{
+  border: 2px solid #141B22; 
+  background-color: #141B22; 
+  color: #fff; 
+  font-size: 16px;  
+  line-height: 24px;
+	gap: 8px;	
+  }
+
+
+  
+.page-header-buttons{
+	display: inline-flex;
+	gap: 20px;
+  }
+
+.welcome-section{
+	background-image: url("../img/binabox-big-pic.svg"), url("../img/star.svg");
+	background-color: #141B22;
+	background-repeat: no-repeat;
+}
+
+.welcome-heading{
+  line-height:80px;
+}
+.welcome-heading-text, .how-heading-text, .partners-text {
+  color: #21E786;
+}
+
+.welcome-text{
+  font-family: "Poppins","Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 30px;
```

Comment:
it is better if the line-height is specified in dimensionless units - in this case line-height: 1.5; (30px / 20px)